### PR TITLE
When using dots, use line too. Make points smaller but still scale with pop.

### PR DESCRIPTION
**Pull request by nmdefries**
I wasn't able to test this due to set-up problems. This should run, but point size may not be ideal.

### CODE REVIEW

**Comment by dsweber2** [comment]
Found the dashboard [example of the points being too large](http://127.0.0.1:3838/?_inputs_&selected_forecasters=%5B%22ensemble_score_advertent.freemasonic%22%2C%22score_sludgy.pleasureseeking%22%2C%22score_appetitive.ribbony%22%2C%22score_barren.photophobic%22%5D&baseline=%22ensemble_score_advertent.freemasonic%22&x_var=%22ahead%22&facet_vars=%22geo_value%22&excluded_geo_values=%5B%22as%22%2C%22gu%22%2C%22vi%22%2C%22mp%22%5D&excluded_aheads=null&selected_forecast_date_range=%5B%222020-01-01%22%2C%222024-02-19%22%5D&selected_target_end_date_range=%5B%222020-01-01%22%2C%222024-02-19%22%5D&selected_metric=%22wis%22&scale_by_baseline=false&facets_share_scale=true&.clientValue-default-plotlyCrosstalkOpts=%7B%22on%22%3A%22plotly_click%22%2C%22persistent%22%3Afalse%2C%22dynamic%22%3Afalse%2C%22selectize%22%3Afalse%2C%22opacityDim%22%3A0.2%2C%22selected%22%3A%7B%22opacity%22%3A1%7D%2C%22debounce%22%3A0%2C%22color%22%3A%5B%5D%7D&plotly_afterplot-A=%22%5C%22main_plot%5C%22%22&plotly_relayout-A=%22%7B%5C%22width%5C%22%3A1052%2C%5C%22height%5C%22%3A1260%7D%22) that we ran into when talking about this:
image
I guess the problem is the point size is relative to the overall interface, rather than the size of the sub-plot? Part of the problem is that the new forecasters have way more points than the old ones, so they get really big in e.g. [this plot](http://127.0.0.1:3838/?_inputs_&selected_forecasters=%5B%22ensemble_score_advertent.freemasonic%22%2C%22score_sludgy.pleasureseeking%22%2C%22score_appetitive.ribbony%22%2C%22score_barren.photophobic%22%2C%22external_scores_55efa869%22%2C%22external_scores_da4d117f%22%2C%22external_scores_3920523c%22%2C%22score_unspecialized.amoebalike%22%5D&baseline=%22external_scores_3920523c%22&x_var=%22ahead%22&facet_vars=null&excluded_geo_values=%5B%22as%22%2C%22gu%22%2C%22vi%22%2C%22mp%22%5D&excluded_aheads=null&selected_forecast_date_range=%5B%222020-01-01%22%2C%222024-02-19%22%5D&selected_target_end_date_range=%5B%222020-01-01%22%2C%222024-02-19%22%5D&selected_metric=%22wis%22&scale_by_baseline=false&facets_share_scale=true&.clientValue-default-plotlyCrosstalkOpts=%7B%22on%22%3A%22plotly_click%22%2C%22persistent%22%3Afalse%2C%22dynamic%22%3Afalse%2C%22selectize%22%3Afalse%2C%22opacityDim%22%3A0.2%2C%22selected%22%3A%7B%22opacity%22%3A1%7D%2C%22debounce%22%3A0%2C%22color%22%3A%5B%5D%7D&plotly_afterplot-A=%22%5C%22main_plot%5C%22%22&plotly_relayout-A=%22%7B%5C%22width%5C%22%3A1052%2C%5C%22height%5C%22%3A1260%7D%22&plotly_hover-A=null)
image

 Either way, I think it'll be useful to make sure that all the kinks are worked out in other people running the dashboard.

The lines are definitely helpful.

**Comment by nmdefries** [comment]
Okay, so maybe scale the point size by # of subplots.

**Comment by dsweber2** [comment]
That did it!
image

**Review by dsweber2 — APPROVED** [review]

File: app.R [hunk 1]
```diff
@@ -262,7 +261,19 @@ shinyApp(
         # Use scatterplot or lines depending on the x var.
         {
           if (input$x_var %in% c(input$facet_vars, "geo_value", "forecaster", "ahead")) {
-            . + geom_point(aes(size = n / 100)) + expand_limits(size = 0)
+            scale_factor_fcast <- length(input$selected_forecasters) * 2
+            scale_factor_facet <- length(input$facet_vars) * 5
+            scale_factor_geo <- ("geo_value" %in% input$facet_vars) * (60 - length(input$excluded_geo_values))
+            scale_factor_geo_x <- ("geo_value" %in% input$x_var) * 10
+            scale_factor_facet_fcast <- ifelse("forecaster" %in% input$facet_vars, length(input$selected_forecasters), 0) * 5
```
Comment (nmdefries) [inline app.R:268]: note: fudge factors for scaling in different situations. A higher number makes all points smaller on the plot.

**Pull request merged**